ui/desktop/src/components/Input.tsx

```diff
@@ -194,11 +197,7 @@ export default function Input({
           type="button"
           size="icon"
           variant="ghost"
-          onClick={(e) => {
```

Comment:
Mind re-inlining this? Lots of our components are getting long with the separate declarations up top and where something can be inlined I think it's a little nicer.